### PR TITLE
[CON-08] Added descriptive assertion texts

### PR DESCRIPTION
**Pull request by mercepluka**

### CODE REVIEW

**Review by ijerkovic — APPROVED** [review]

File: contracts/bridge_token_deployer/src/lib.rs [hunk 1]
```diff
@@ -89,7 +92,7 @@ impl BridgeTokenDeployer {
 
     #[private]
     pub fn complete_deployment(&mut self, bridge_token_address: AccountId) {
-        require!(env::promise_results_count() == 1);
+        require!(env::promise_results_count() == 1, "Was Expecting one and only one result");
```
Comment (medicz) [inline contracts/bridge_token_deployer/src/lib.rs:95]:
```suggestion
        require!(env::promise_results_count() == 1, "One and only one result was expected");
```

File: contracts/connector_base/src/macros.rs [hunk 2]
```diff
@@ -6,7 +6,7 @@ macro_rules! impl_deployer_aware {
             #[payable]
             #[private]
             fn set_deployer(&mut self, deployer_account: AccountId) {
-                require!(self.deployer_account.is_none());
+                require!(self.deployer_account.is_none(), "Deployer account should be none");
```
Comment (medicz) [inline contracts/connector_base/src/macros.rs:9]:
```suggestion
                require!(self.deployer_account.is_none(), "Deployer account should not be set");
```

File: contracts/connector_base/src/macros.rs [hunk 3]
```diff
@@ -52,7 +52,7 @@ macro_rules! impl_deployer_aware {
 
             #[private]
             fn complete_deployment(&mut self, source_address: AccountId) {
-                require!(env::promise_results_count() == 1);
+                require!(env::promise_results_count() == 1, "Was Expecting one and only one result");
```
Comment (medicz) [inline contracts/connector_base/src/macros.rs:55]:
```suggestion
                require!(env::promise_results_count() == 1, "One and only one result was expected");
```

File: contracts/connector_base/src/macros.rs [hunk 4]
```diff
@@ -141,7 +141,7 @@ macro_rules! impl_other_network_token_aware {
                 destination_contract: AccountId,
                 proof: FullOutcomeProof,
             ) {
-                require!(env::promise_results_count() == 1);
+                require!(env::promise_results_count() == 1, "Was Expecting one and only one result");
```
Comment (medicz) [inline contracts/connector_base/src/macros.rs:144]:
```suggestion
                require!(env::promise_results_count() == 1, "One and only one result was expected");
```

File: contracts/connector_base/src/macros.rs [hunk 5]
```diff
@@ -165,7 +165,7 @@ macro_rules! impl_other_network_token_aware {
 
             #[payable]
             fn register_on_other(&mut self, proof: FullOutcomeProof, height: u64) {
-                require!(self.locker_account.is_some());
+                require!(self.locker_account.is_some(), "Locker account should not be none");
```
Comment (medicz) [inline contracts/connector_base/src/macros.rs:168]:
```suggestion
                require!(self.locker_account.is_some(), "Locker account should be set");
```

File: contracts/connector_base/src/macros.rs [hunk 6]
```diff
@@ -222,7 +222,7 @@ macro_rules! impl_other_network_aware {
             #[payable]
             #[private]
             fn set_locker(&mut self, locker_account: AccountId) {
-                require!(self.locker_account.is_none());
+                require!(self.locker_account.is_none(), "Locker account should be none");
```
Comment (medicz) [inline contracts/connector_base/src/macros.rs:225]:
```suggestion
                require!(self.locker_account.is_none(), "Locker account should not be set");
```

File: contracts/connector_base/src/macros.rs [hunk 7]
```diff
@@ -277,7 +277,7 @@ macro_rules! impl_token_mint {
             #[payable]
             fn mint(&mut self, proof: FullOutcomeProof, height: u64) {
                 self.assert_not_paused(PAUSE_MINT);
-                require!(self.locker_account.is_some());
+                require!(self.locker_account.is_some(), "Locker account should not be none");
```
Comment (medicz) [inline contracts/connector_base/src/macros.rs:280]:
```suggestion
                require!(self.locker_account.is_some(), "Locker account should be set");
```

File: contracts/ft_connector/src/lib.rs [hunk 8]
```diff
@@ -110,7 +110,7 @@ impl FungibleTokenConnector {
         ft_contract_id: AccountId,
         amount: U128,
     ) {
-        require!(env::promise_results_count() == 1);
+        require!(env::promise_results_count() == 1, "Was Expecting one and only one result");
```
Comment (medicz) [inline contracts/ft_connector/src/lib.rs:113]:
```suggestion
        require!(env::promise_results_count() == 1, "One and only one result was expected");
```

File: contracts/xsc_connector/src/lib.rs [hunk 9]
```diff
@@ -141,7 +141,7 @@ impl CrossShardConnector {
 
     #[payable]
     pub fn cross_call_execute(&mut self, proof: FullOutcomeProof, height: u64) {
-        require!(self.locker_account.is_some());
+        require!(self.locker_account.is_some(), "Locker account should not be none");
```
Comment (medicz) [inline contracts/xsc_connector/src/lib.rs:144]:
```suggestion
        require!(self.locker_account.is_some(), "Locker account should be set");
```

File: contracts/xsc_connector/src/lib.rs [hunk 10]
```diff
@@ -258,7 +258,7 @@ impl CrossShardConnector {
         source_contract_method: String,
         destination_contract: AccountId,
     ) {
-        require!(env::promise_results_count() == 1);
+        require!(env::promise_results_count() == 1, "Was Expecting one and only one result");
```
Comment (medicz) [inline contracts/xsc_connector/src/lib.rs:261]:
```suggestion
        require!(env::promise_results_count() == 1, "One and only one result was expected");
```

File: contracts/xsc_connector/src/lib.rs [hunk 11]
```diff
@@ -273,7 +273,7 @@ impl CrossShardConnector {
 
     #[payable]
     pub fn cross_call_receive_response(&mut self, proof: FullOutcomeProof, height: u64) {
-        require!(self.locker_account.is_some());
+        require!(self.locker_account.is_some(), "Locker account should not be none");
```
Comment (medicz) [inline contracts/xsc_connector/src/lib.rs:276]:
```suggestion
        require!(self.locker_account.is_some(), "Locker account should be set");
```

File: contracts/xsc_connector/src/lib.rs [hunk 12]
```diff
@@ -332,7 +332,7 @@ impl CrossShardConnector {
         response: String,
         proof: FullOutcomeProof,
     ) {
-        require!(env::promise_results_count() == 1);
+        require!(env::promise_results_count() == 1, "Was Expecting one and only one result");
```
Comment (medicz) [inline contracts/xsc_connector/src/lib.rs:335]:
```suggestion
        require!(env::promise_results_count() == 1, "One and only one result was expected");
```

**Review by medicz — CHANGES_REQUESTED** [review]
improve wording of errors

**Review by medicz — APPROVED** [review]

**Pull request merged**